Add changelog for 2.0.0

consideRatio opened the pull request:
I think this will result in closing #53, otherwise it will just make us use more updated node dependencies which seems fine as well. I did not bump any dependency a major version (none was around) so I think this will be fine to merge as a non-major release.

consideRatio committed:
Add changelog for 2.0.0

manics commented:
Nodejs is up to date in the build step, but it also needs to be updated in the runtime specification https://github.com/jupyterhub/action-k8s-await-workloads/pull/60

consideRatio commented:
@manics I updated this PR to include #60 and made it 2.0.0 instead of 1.1.0! Thanks for helping out with this!!!

I also added a commit updating the README.md example and the associated github test job to verify it still works. That will keep failing until we have made this release.

consideRatio committed:
docs: update README.md for v2 release

manics reviewed (approved):
LGTM!
